README.md

```diff
@@ -1 +1,70 @@
-# payment-button-elementor-plugin
+=== Razorpay Payment Button Elementor Plugin ===
+Contributors: razorpay
+Tags: razorpay, payments, india, ecommerce
+Tested up to: 1.1
+Stable tag: 1.1
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+== Description ==
+
+A very simple elementor widget block native to the wordpress that lets you add a convenient button on your wordpress page. This widget is like any other wordpress plugin and can be found immediately in the widgets section.
+
+These are the features that you will get without any premium paywalls
+1. Collect payments for multiple product you want to sell
+2. No hops - your customer pays on the same page and doesn’t get redirected to a different page
+3. No extra login required for the customer to pay
+4. Fully editable form fields
+5. Logo and brand colour of your liking
+6. Collect payments in 100 currencies
+7. 4 button styles
+8. FREE payment receipt that goes to your customers
+9. Highest security standards in PCI-DSS Compliance
+
+This will enable to collect payments in multiple payment methods such as -
+* Credit Card,
+* Debit Card
+* Netbanking
+* Wallets
+* UPI / QR 
+
+== Installation ==
+
+1. Install the plugin from the [Wordpress Plugin Directory].
+2. Navigate to Razorpay Buttons Elementor → Settings to enter configuration details of Razorpay.
+3. Navigate to Razorpay Buttons Elementor and check the list of buttons available on Razorpay.
+4. Navigate to Pages → All Pages/Add New to embed Payment button to WordPress page.
+5. Press the ‘Edit with Elementor’ button.
+   Now you can drag and drop widgets from the left panel onto the content area, as well as add new sections and columns that make up the page structure
```

Comment:
@neera11 please add where the Razorpay Payment Button widget will be available on the widget section.